src/styles/day.scss

```diff
@@ -0,0 +1,28 @@
+@import './index.scss';
+
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  background-color: #eee;
+  border: 1px solid #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: $font-family;
+  position: relative;
+
+  // Показ числа
+  &::before {
+    content: attr(data-day);
+    font-size: 30px;
+    color: $primary-color;
```

Comment:
Ensure that the variable `$primary-color` is defined in the `index.scss` file or another imported SCSS file. Otherwise, this will cause an error during compilation.